src/common/storage/src/parquet_rs.rs

```diff
@@ -131,6 +140,50 @@ pub async fn read_metadata_async(
     }
 }
 
+pub fn read_metadata_sync(
```

Comment:
We can consider using https://crates.io/crates/parquet_opendal

Reply:
Yes, but this is for sync api yet parquet_opendal only support async api.

Reply:
> Yes, but this is for sync api yet parquet_opendal only support async api.

Would you like to create an issue for this?